app/src/main/java/com/globant/practice/domain/service/GitHubApi.java

```diff
@@ -0,0 +1,47 @@
+package com.globant.practice.domain.service;
+
+import com.globant.practice.domain.model.Profile;
+import com.globant.practice.domain.model.Repository;
+import com.globant.practice.domain.model.User;
+import java.util.List;
+import io.reactivex.Observable;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+import retrofit2.http.Url;
+
+/**
+ * Interface that is required by retrofit and contains all the possible api calls that the
+ * application need.
+ * Created by jonathan.vargas on 4/04/2017.
+ */
+
+public interface GitHubApi {
+
+    /**
+     * Api call that returns a list of the Github users.
+     *
+     * @return
+     */
+    @GET
+    Observable<List<User>> getUsers(@Url String url);
```

Comment:
@Url is commonly used for Dynamic URLs. Why don't you define *repos/{login}/{repo}/subscribers* inside the GET and receive the login (googlesamples) and repo (android-architecture) as parameters?